config/wof.yml

```diff
@@ -0,0 +1,16 @@
+site_name: Who's On First API
+docs_dir: src-wof
+site_dir: dist-wof
+
+pages:
+ - Home: index.md
+ - 'Formats': 'formats.md'
+ - 'Methods': 'methods.md'
+ - 'Pagination': 'pagination.md'
+ - 'Error codes': 'errors.md'
+
+extra:
+ site_subtitle: "Retrieve data about places from our gazetteer, Who's On First."
```

Comment:
Rather than use our here, how about, "Retrieve data about places from the Who's on First gazetteer.

Retrieve is a little generic, but good enough.